File: revisao/revisao01/DescontoEmCompras.java
import java.util.Scanner;

class DescontoEmCompra{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);

        double valorDaCompra = 0;
        
        valorDaCompra = scan.nextDouble();

//TODO: Adicione uma verificação se o valor da compra inserido não foi menor ou igual a 0,
    //  caso seja menor ou igual a zero informe que não será possível processar, do contrário continue a verificação
        if(valorDaCompra<100){
            System.out.println("Não é aplicado desconto em compras com valor menor que 100");
            System.out.println("Você deverá pagar: R$"+String.format("%.2f",valorDaCompra));
        } else if(valorDaCompra >= 100 && valorDaCompra <200){
            System.out.println("Hmm, você tem direito a um desconto de 10% (Você pagará 90% do valor da compra)");
            System.out.println("Você deverá pagar: R$"+String.format("%.2f",valorDaCompra * 0.9));
        } else if(valorDaCompra >=200){
            System.out.println("Hmm, você tem direito a um desconto de 20% (Você pagará 80% do valor da compra)");
//TODO: Atribua o cálculo do desconto a variável valorComDesconto e faça com que ele seja impreso no lugar do valor 0.0
            double valorComDesconto;
            System.out.println("Você deverá pagar: R$"+String.format("%.2f", 0.0));
        }
    }
}

File: revisao/revisao01/DescontoEmCompras.py
valor_da_compra = float(input('Insira o valor da compra:'))

#TODO: Adicione uma verificação se o valor da compra inserido não foi menor ou igual a 0,
#         caso seja menor ou igual a zero informe que não será possível processar, do contrário continue a verificação
if valor_da_compra <100:
    print('Não há descontos aplicáveis para sua compra.')
    print(f'O valor do compra é de {valor_da_compra:.2f}')
elif valor_da_compra >= 100 and valor_da_compra<200:
    print('Opaa! 10% de descontos aplicáveis à sua compra.')
    print(f'O valor do compra é de {valor_da_compra*0.9:.2f}') # 0.9 = 90% (Lembre que o uso do símbolo da porcetagem é usada pega cálcular o resto de uma divisão)
elif valor_da_compra >200:
    print('Opaa! 20% de descontos aplicáveis à sua compra.')
# Corrija o valor do desconto aplicado
    print(f'O valor do compra é de {valor_da_compra * 0.0:.2f}')